packages/storybook/examples/core/TextLabel.stories.js

```diff
@@ -0,0 +1,181 @@
+                    status="success"
+                    statusOptions={{ autohide: false }} />
+            </DebugBox>
+
+            <DebugBox>
+                <TextLabel
+                    icon="printer"
+                    basic="Error"
+                    tag="Tag"
+                    aside="Left align"
+                    status="error"
+                    errorMsg="Save failed" />
+            </DebugBox>
+        </div>
+    );
+}
+
+export function TextLabelWithTextExample() {
+    return (
+        <div>
+            <DebugBox>
+                <TextLabel>
+                    <Text basic="foo" tag="Tag" />
+                </TextLabel>
+            </DebugBox>
+
+            <DebugBox>
+                <TextLabel status="loading">
+                    <Text basic="foo" aside="loading..." />
+                </TextLabel>
+            </DebugBox>
+        </div>
+    );
+}
+TextLabelWithTextExample.story = {
+    name: '<Text> in child',
+};
+
+/**
+ * The official `action` shipped with React Storybook crashes when
+ * your payload contains an DOM node.
+ *
+ * This decorated action stripes DOM nodes, only shows their names.
+ */
+const cleanAction = decorateAction([
+    ([payload]) => {
+        const cleanPayload = {
+            ...payload,
+            event: `[${payload.event.constructor.name}]`,
+        };
+        return [cleanPayload];
+    }
+]);
+export function UncontrolledEditableTextLabel() {
+    return (
+        <div>
+            <EditableTextLabel
+                icon="printer"
+                basic="Kitchen Printer"
+                aside="00:11:22:33"
+                tag="Online"
+                onDblClick={action('dblClick')}
+                onEditEnd={cleanAction('editEnd')} />
+        </div>
+    );
+}
+
+export function ControlledEditableTextLabel() {
+    const [isEditing, setIsEditing] = useState(false);
+    const [status, setStatus] = useState(null);
+    const [currentBasic, setCurrentBasic] = useState('Kitchen Printer');
+
+    const handleDblClick = (event) => {
+        setIsEditing(true);
+        action('dblClick')(event);
+    };
+
+    const handleEditEnd = (payload) => {
+        // When value isn't `null`, it's not a cancelling action from ESC key
+        const hasValue = payload.value !== null;
+
+        if (hasValue) {
+            setCurrentBasic(payload.value);
+            setTimeout(() => setStatus('success'), 600);
+        }
+        setIsEditing(false);
+        setStatus(hasValue ? 'loading' : null);
+        cleanAction('editEnd')(payload);
+    };
+
+    return (
+        <DebugBox>
+            <EditableTextLabel
+                inEdit={isEditing}
+                icon="printer"
+                basic={currentBasic}
+                aside="00:11:22:33"
+                tag="Online"
+                onDblClick={handleDblClick}
+                onEditEnd={handleEditEnd}
+                status={status} />
+        </DebugBox>
+    );
+}
```

Comment:
We move old example https://github.com/iCHEF/gypcrete/pull/252/files#diff-686ccfa81e5d6298db6dc1b22096b900 to here. Note that we tear `Editable` example apart (controlled and non-controled) to make it easier to read.

Reply:
I was thinking maybe we should put those editable examples in their own doc page?
Because they are quite different from `<TextLabel>`s and they take different props.

Reply:
Sounds good let's do it

Reply:
Done